concordium-sdk/src/main/java/com/concordium/sdk/responses/branch/Branch.java

```diff
@@ -0,0 +1,38 @@
+package com.concordium.sdk.responses.branch;
+
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.transactions.Hash;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.*;
+
+import java.util.List;
+
+@Data
```

Comment:
I'm not sure why you also annotate with `@ToString` and `@RequiredArgsConstructor` as `@Data` already puts these on the type.

Reply:
removed